File: haskell/1002.hs

-- https://www.urionlinejudge.com.br/judge/pt/problems/view/1002

import Text.Printf (printf)

areaCirculo :: Double -> Double
areaCirculo raio = 3.14159 * (raio ^ 2)

main :: IO ()
main = do a <- readLn :: IO Double
          printf "A=%.4g\n" (areaCirculo a)

File: c/1002.c

// https://www.urionlinejudge.com.br/judge/pt/problems/view/1002

#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main()
{
    double r, pi, a;
    pi = 3.14159;

    scanf("%lf", &r);

    a = pi * pow(r,2);

    printf("A=%.4lf\n", a);

    return 0;
}